Can't build provider on my local machine

Hey,
thanks for your provider. I get interested in and want to contribute if I can. So I tried to build locally and I got the following stuff.


## What I did ?

```
sudo npm i -g swagger swagger-codegen
make
```

## Output

```sh
scripts/generate-client.sh
Remove old generated sources

error: invalid command: generate
Usage: swagger [options] [command]

Options:
  -V, --version                                         output the version number
  -h, --help                                            output usage information

Commands:
  project <action>                                      project actions
  docs                                                  open Swagger documentation
  validate [options] [swaggerFile]                      validate a Swagger document (supports unix piping)
  convert [options] <swaggerFile> [apiDeclarations...]  Converts Swagger 1.2 documents to a Swagger 2.0 document
  help [cmd]                                            display help for [cmd]
rm -f ./terraform-provider-harbor_*
echo "Building terraform-provider-harbor_1.0_linux_amd64..."
Building terraform-provider-harbor_1.0_linux_amd64...
env GOOS=linux GOARCH=amd64 go build -o terraform-provider-harbor_v1.0_linux_amd64 .
go: finding module for package github.com/nolte/terraform-provider-harbor/gen/harborctl/client/products
go: finding module for package github.com/nolte/terraform-provider-harbor/gen/harborctl/client
go: finding module for package github.com/nolte/terraform-provider-harbor/gen/harborctl/models
client/client.go:10:2: no matching versions for query "latest"
harbor/data_source_project.go:7:2: no matching versions for query "latest"
harbor/resource_config_auth.go:11:2: no matching versions for query "latest"
make: *** [Makefile:26: build] Error 1
```
Let me know if I can contribute anything useful to the project.

--- a/README.md
+++ b/README.md
@@ -17,9 +17,24 @@ and open [127.0.0.1:8000](http://127.0.0.1:8000/)
 
 ## Building
 
+## Visual Studio Code DevContainer
+
 For Easy development use [Visual Studio Code DevContainer](https://code.visualstudio.com/docs/remote/containers), you can find the basement from the Development Containers at [nolte/vscode-devcontainers](https://github.com/nolte/vscode-devcontainers).
 
-```bash
+1. Create you Github Personal Access Token under https://github.com/settings/tokens with the following scopes:
+   1. `read:packages`
+
+2. Login to fetch the required dev containers
+
+```sh
+docker login docker.pkg.github.com
+```
+
+4. Grab you a Coffee and wait for 3 Minutes (This happens on the first time use)
+
+3. Click Terminal -> New Terminal and execute the following command:
+
+```sh
 # using the Makefile
 make
 ```

